lib/graphql/static_validation/rules/input_object_names_are_unique_error.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module GraphQL
+  module StaticValidation
+    class InputObjectNamesAreUniqueError < StaticValidation::Error
+      attr_reader :name
+
+      def initialize(message, path: nil, nodes: [], name:)
+        super(message, path: path, nodes: nodes)
+        @name = name
+      end
+
+      # A hash representation of this Message
+      def to_h
+        extensions = {
+          "code" => code,
+          "name" => name
+        }
+
+        super.merge({
+          "extensions" => extensions
+        })
+      end
+
+      def code
+        "inputFieldNotUnique"
```

Comment:
I couldn't find a precedent for these `code`s in the JS implementation: https://github.com/graphql/graphql-js/search?q=argumentNotUnique

Reply:
LGTM :+1: